app/views/users/show.html.erb

```diff
@@ -1,40 +1,41 @@
-
 <%- model_class = User -%>
 
 <div class="page-header">
   <h1><%=t '.title' %></h1>
 </div>
 
-<% if @user == current_user %>
-  <div class="panel panel-default">
-    <div class="panel-heading">
-      <div class="row">
-        <div class="col-sm-6">
-          <h4><%= t('.user_data')%></h4>
-        </div>
-        <div class="col-sm-6 text text-right">
-          <%= link_to t('.edit', default: t('helpers.links.edit')), edit_user_path, class: 'btn btn-primary' %>
-        </div>
+<div class="panel panel-default">
+  <div class="panel-heading">
+    <div class="row">
+      <div class="col-sm-6">
```

Comment:
Du verwendest hier ja 'col-sm-6'. Ich glaube in unseren Fall wäre es auch ganz gut  'col-md-6' zu nehmen, da wir ja diese Applikation hauptsächlich für den Desktop gebrauch nehmen :) (macht nicht viel aus, wäre aber dann auch einheitlicher)